README.md

```diff
@@ -1,35 +1,34 @@
-# Districtr Reboot
+<img src="docs/images/districtr-logo.svg" alt="Districtr logo" width="150"/>
 
 The Districtr reboot monorepo.
 
-## Repo structure
+## Repo organization
 
-- [`app`](app/): The rebooted Districtr app, NextJS/React Typescript app
-- [`backend`](backend/): Backend. TBD tech stack
-- [`pipelines`](pipelines/): Data pipelines, ETL. Thinking of using Dagster.
+- [`app`](app/): The rebooted Districtr app, NextJS/React Typescript app.
+- [`backend`](backend/): FastAPI/Python API.
+- [`pipelines`](pipelines/): Data pipelines, ETL. Not a main focus of the reboot. For now, will mostly contain scratch data transformation scripts before being integrated into the backend CLI.
 - [`prototypes`](prototypes/): Prototypes conducted as part of the reboot.
 
-### App
+## Districtr reboot architecture
 
-TK
+After experimenting with various technologies (see [`prototypes`](prototypes/)) we landed on the following architecture for the Districtr reboot:
 
-### Backend
+![Districtr architecture](docs/images/districtr-architecture.png "Districtr architecture")
 
-TK
+The redesign aims to principally to address three key pain points in the Districtr application’s performance and maintainability:
+1. Slow tile rendering
+1. Cumbersome use of tiles as global state for tile rendering and most metric calculation
+1. Complexity and poor interoperability in architecture without slow copies
 
-### Pipelines
+And two key feature additions
+1. Block “shattering”
+1. A headless CMS
```

Comment:
should be "2" and the CMS is out of our scope right? if so we should indicate that it is for a future phase imo

Reply:
Will add a note that CMS is out of scope for now. As for the numbering, MD is smart and will number lists automatically so you don't need to worry about always keeping the numbers ordered.

<img width="1193" alt="image" src="https://github.com/user-attachments/assets/31b53833-5aa7-4318-858a-ca2d43669dea">